classes/class06/sessions/app.js

```diff
@@ -0,0 +1,33 @@
+var express = require('express');
+var session = require('express-session');
+var path = require('path');
+
+var logger = require('morgan');
+var bodyParser = require('body-parser');
+var cookieParser = require('cookie-parser');
+
+var app = express();
+
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(session({
+	secret: 'secret',
+	resave: false,
+	saveUninitialized: true
+}));
+
+app.get('/', function(req, res) {
+	console.dir(req.cookies);
```

Comment:
http://nodejs.org/api/console.html#console_console_log_data

Console.log can take multiple arguments and can use string formatting, but a single argument call to log and dir both use util.inspect() the same way.